manifests/cf/cell.yml

```diff
@@ -2,23 +2,11 @@
 instance_groups:
 - name: cell
   jobs:
-  - name: consul_agent
-    release: consul
-    consumes:
-      consul_common: {from: consul_common_link}
-      consul_server: nil
-      consul_client: {from: consul_client_link}
 
   - name: cflinuxfs2-rootfs-setup
     release: cflinuxfs2
-    properties:
-      cflinuxfs2-rootfs:
-        trusted_certs: (( grab meta.certs.consul.application_ca ))
   - name: cflinuxfs3-rootfs-setup
     release: cflinuxfs3
-    properties:
-      cflinuxfs3-rootfs:
-        trusted_certs: (( grab meta.certs.consul.application_ca ))
```

Comment:
I get that consul is gone, but what was the role that the `application_ca` property was playing, and has that role moved elsewhere?